lte/cloud/go/services/subscriberdb/servicers/southbound/subscriberdb_servicer.go

```diff
@@ -110,12 +111,18 @@ func (s *subscriberdbServicer) Sync(
 		return &lte_protos.SyncResponse{Resync: true}, nil
 	}
 
-	resync, renewed, deleted, err := s.getSubscribersChangeset(networkID, req.LeafDigests, digestTree.LeafDigests)
+	resync, renewed, deleted, err := s.getSubscribersChangeset(ctx, networkID, req.LeafDigests, digestTree.LeafDigests)
```

Comment:
GetCloudSubscriberDbEnabled(ctx) is a bit heavy, it has to lookup GW info, load network configs (all DB lookups/reads). I think it'd make sense to pass cloudflag bool to getSubscribersChangeset as a parameter instead of making another redundant call to GetCloudSubscriberDbEnabled() inside getSubscribersChangeset().